documentation/data-importer.adoc

```diff
@@ -138,6 +136,28 @@ If you're unsure if you've followed the steps correctly so far, you can go ahead
 button::Load Northwind node mapping[role=NX_IMPORT_LOAD,endpoint={northwind-subset-node-only-mapping-zip-file}]
 ====
 
+== Constraints and indexes
+
+To ensure accuracy of your data and support read performance, Data Importer automatically creates a uniqueness constraint and a corresponding index on the property you select as the node ID.
+You can't modify the two that are created for you, but you can add additional indexes.
+
+Constraints and indexes have their own tab in the highlight:import/import-mapping-panel[mapping details panel].
```

Comment:
This feels like it will be a big block of text and probably difficult to read. Could you split into paragraphs.

Maybe add the `Only the properties you mapped from the _Definition_ tab are available.` sentence as a [NOTE] at the end.

Reply:
✅